unit_tests/tests/actuators/test_etb.cpp

```diff
@@ -199,16 +200,25 @@ TEST(etb, initializationNoPrimarySensor) {
 
 	EtbController dut;
 
-	// Needs pedal for init
-	Sensor::setMockValue(SensorType::AcceleratorPedal, 0.0f, true);
+	EngineTestHelper eth(TEST_ENGINE, [](engine_configuration_s* engineConfiguration) {
+	});
 
-	// Redundant, but no primary configured
-	Sensor::setMockValue(SensorType::Tps1, 0.0f, true);
+	engineConfiguration->tps1_1AdcChannel = EFI_ADC_NONE;
```

Comment:
we should never be setting pins in a unit test like this, there are explicit TPS init tests, and this is not one of them